src/doc/gui_manual/Operators/OperatorTypes/PICS/IntegralCurve_operator.rst

```diff
@@ -0,0 +1,298 @@
+.. _Integral Curve operator:
+
+Integral Curve operator
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Source
+^^^^^^
+
+Source type
+"""""""""""
+
+The user can seed the integral curves using a single point; an arbitrary list of points; samples on a line, plane, circle, sphere, or box; a Named Selection; or Field Data. The following is a list of source types and their respective parameters.
```

Comment:
I wanna ask about the thinking cap to wear when documenting details like this. The way I see it, this sentence essentially parrots what one would actually see in the GUI. If a new source type is added to the code, we need to update this sentence to include it. So, this is increasing the maintenance burden. Where as, if documentation was crafted slightly differently...

> The source type controls how the *seeds* for curves are created. There are various options the names of which are self-descriptive such as creating them along a *line* or around a *sphere*. Only those options that require further clarification are described further here...

Would this allow you to remove most of the sections below which, lets be honest, are describing things we would expect most users to already understand

Reply:
I agree that it would be nice to have less doc burden on things that are obvious like this. It's difficult for me to know how much knowledge to assume a user has. What do you suggest is a good rule of thumb?

Reply:
Hmm...would *gut feel* be a reasonable answer? 😄 Honestly though, I am not sure I have a good answer about that...

- I think we should be compelled to write documentation for those things in the interface when the interface itself is not self-descriptive enough for a *prototypical user* (we should probably spend some a moment to define what that is) to *get it* without further clarification.
- We could take a *successive approximation* approach and write only the minimal documentation we think is needed and only add to it when/if we get (multiple) inquiries from users.
- I do believe the better the interface is, the less documentation should be required to support it.
- I wonder if having more *examples* (of course keeping those maintained is a problem too), is *better* than formal docs?

I don't think we should expend effort writing documentation for the least able user and I don't think we should assume all users are experts either.

Reply:
> Hmm...would _gut feel_ be a reasonable answer?

Ya, I think this is reasonable. With reviewers, we can get the combined "gut feel" of several developers.

> successive approximation approach

I also like this idea as well.